docs/streams/core-concepts.html

```diff
@@ -127,6 +127,10 @@ <h3><a id="streams_time" href="#streams_time">Time</a></h3>
         <li> For aggregations, the timestamp of a resulting aggregate update record will be that of the latest arrived input record that triggered the update.</li>
     </ul>
 
+    <p>
+	Note, that the describe default behavior can be changed in the Processor API by assigning timestamps to output records explicitly when calling <code>#forward()</code>.
```

Comment:
Note users can also explicitly override the default assigned timestamp of the generated results when sending to the downstream processor via <code>#forward()</code>

Reply:
Yes. It seems to be covered already in L125.